src/app/js/public/ExportShare.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import compose from 'recompose/compose';
+import { connect } from 'react-redux';
+import translate from 'redux-polyglot/translate';
+import Divider from 'material-ui/Divider';
+
+import { polyglot as polyglotPropTypes } from '../propTypes';
+import { fromCharacteristic } from './selectors';
+import { fromFields } from '../sharedSelectors';
+import AppliedFacetList from './facet/AppliedFacetList';
+import Export from './export/Export';
+import Share from './Share';
+import ShareLink from './ShareLink';
+import Widgets from './Widgets';
+import Version from './Version';
+
+export const ExportShareComponent = ({
+    sharingUri,
+    uri,
+    sharingTitle,
+    p: polyglot,
+}) => (
+    <div>
+        <AppliedFacetList />
+        <Export uri={uri} />
```

Comment:
seeing this, I find that the `Export` component is badly named. It's an `Export`-what? A button ? A utility ? "Export" is a verb, we usually use names for components.